### PR TITLE
chore(IT Wallet): [SIW-1378] Remove image claim decoding without URL schema

### PR DESCRIPTION
**Pull request by LazyAfternoons**
## Short description
This PR removes the workaround for decoding image claims that lack a proper URL schema. It also adds support for multiple types of images according to the mapped [Image](https://reactnative.dev/docs/image#source) component. 
`gif`, `webp`, `psd` are not included because they are either platform specific (`psd`) or require additional setup (`gif` and `webp` on Android).

## List of changes proposed in this pull request
- Remove the `ImageClaimNoUrl` decoder and its usage; 
- Add supports for multiple image formats (`png`, `jpg`, `jpeg`, `bmp`);
- Add tests for the `ImageClaim` decoder.

## How to test
Static tests and check if the image claim is rendered properly after the issuer adds the URL schema.

### CODE REVIEW

File: ts/features/itwallet/common/utils/itwClaimsUtils.ts [hunk 1]
```diff
@@ -162,15 +162,7 @@ const DATE_FORMAT_REGEX = "^\\d{4}-\\d{2}-\\d{2}$";
...
-const PICTURE_URL_REGEX = "^data:image\\/png;base64,";
-
-/**
- * Regex for the picture without URL format which is used to validate the image claim as a base64 encoded png image.
- * This is needed until the issuer adds the URL to the image claim.
- * TODO [SIW-1378]: remove this regex when the issuer adds the URL schema to the image claim.
- */
-const PICTURE_WITHOUT_URL_REGEX =
-  "(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)";
+const PICTURE_URL_REGEX = "^data:image\\/(png|jpg|jpeg|bmp|gif|webp);base64,";
```
Comment (gispada) [inline ts/features/itwallet/common/utils/itwClaimsUtils.ts:165]: In the PR description you said `gif` and `webp` were not included?

File: ts/features/itwallet/common/utils/__tests__/itwClaimsUtils.test.ts [hunk 2]
```diff
@@ -143,3 +145,36 @@ describe("extractFiscalCode", () => {
...
+
+describe("ImageClaim", () => {
+  const base64 =
+    "iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAIAAACQd1PeAAAAEElEQVR4nGKKrzEGBAAA//8CVAERMRFlewAAAABJRU5ErkJggg==";
+  it("should decode a valid png image", () => {
+    const decoded = ImageClaim.decode(`data:image/png;base64,${base64}`);
+    expect(E.isRight(decoded)).toBe(true);
+  });
+  it("should decode a valid jpg image", () => {
+    const decoded = ImageClaim.decode("data:image/jpg;base64,${base64}");
```
Comment (gispada) [inline ts/features/itwallet/common/utils/__tests__/itwClaimsUtils.test.ts:157]: Shouldn't you use backticks to interpolate `${base64}`?

Comment (LazyAfternoons) [reply ts/features/itwallet/common/utils/__tests__/itwClaimsUtils.test.ts:157]: There was a commit I did not push, sorry.

File: ts/features/itwallet/common/utils/itwClaimsUtils.ts (hunk 1 above)
Comment (LazyAfternoons) [reply ts/features/itwallet/common/utils/itwClaimsUtils.ts]: There was a commit I did not push, sorry.

**Review by hevelius — APPROVED** [review]

**Pull request merged**